docker-bake.hcl

```diff
@@ -21,7 +21,7 @@ target "binaries" {
 }
 
 target "all-arch" {
-  platforms = ["linux/amd64", "linux/arm64", "linux/arm", "linux/ppc64le", "linux/s390x", "linux/riscv64", "linux/386"]
+  platforms = ["linux/amd64", "linux/arm64", "linux/arm", "linux/ppc64le", "linux/s390x", "linux/riscv64", "linux/386", "linux/mips64el"]
```

Comment:
Error is
```
#10 [linux/mips64el internal] load metadata for docker.io/library/base-mips64el:latest
#10 sha256:ac77c6a20c5733238141da74d482e6cbc88691987e8b76744ff534d439afa53b
#10 ERROR: pull access denied, repository does not exist or may require authorization: server message: insufficient_scope: authorization failed
```
Don't add mips in this file as releasing image for mips is more complicated due to missing base images etc. and not that important. It's good enough just to build mips emulator for other platforms.

Reply:
@tonistiigi Thank you very much